grpc/runtime/src/main/scala/io/buoyant/grpc/runtime/DecodingStream.scala

```diff
@@ -30,7 +31,7 @@ private[runtime] trait DecodingStream[T] extends Stream[T] {
    * This state holds a `Releaser`, which manages when HTTP/2 frames
```

Comment:
Is this still considered a state of the stream or is it now just functioning as a buffer? Maybe the comments need to be updated?

Reply:
it's still state and it still holds a releaser, even if it's less state than we were storing before.